all-actors/src/main/java/org/sunbird/serviceimpl/CertsServiceImpl.java

```diff
@@ -0,0 +1,177 @@
+package org.sunbird.serviceimpl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mashape.unirest.http.HttpResponse;
+import com.mashape.unirest.http.JsonNode;
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Logger;
+import org.sunbird.BaseException;
+import org.sunbird.CertVars;
+import org.sunbird.JsonKeys;
+import org.sunbird.builders.Certificate;
+import org.sunbird.builders.Course;
+import org.sunbird.message.IResponseMessage;
+import org.sunbird.message.ResponseCode;
+import org.sunbird.request.Request;
+import org.sunbird.response.Response;
+import org.sunbird.service.ICertService;
+import org.sunbird.utilities.CertificateUtil;
+
+import java.text.MessageFormat;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Future;
+
+public class CertsServiceImpl implements ICertService {
+    static Logger logger = Logger.getLogger(CertsServiceImpl.class);
+    private static ObjectMapper requestMapper = new ObjectMapper();
+
+    @Override
+    public String add(Request request) throws BaseException {
+        Map<String, Object> certAddReqMap = new HashMap<>();
+        certAddReqMap = request.getRequest();
```

Comment:
no need of above line : 
certAddReqMap can be Map<String, Object> certAddReqMap = request.getRequest();

Reply:
done